static/styles.css

```diff
@@ -0,0 +1,46 @@
+body {
```

Comment:
Move to `templates/static`

Reply:
Thank you for the feedback! I will move the `styles.css` file to a new `templates/static` directory and update the paths accordingly.

I'll make the necessary changes and update the PR.